app/views/campaigns/show.html.erb

```diff
@@ -20,14 +20,7 @@
       <h2 class="nhsuk-heading-s">Save offline</h2>
       <p>Record vaccinations while not connected to the internet.</p>
 
-      <form method="post">
-        <button class="nhsuk-button nhsuk-u-margin-bottom-3"
-                type="submit"
-                data-action="offline-campaign#saveOffline:prevent"
-                data-testid="save-offline">
-          Save offline
-        </button>
-      </form>
+      <a href="<%= setup_offline_path %>" class="nhsuk-button nhsuk-u-margin-bottom-3" data-testid="save-offline">Save offline</a>
```

Comment:
With the action connection to `offline-campaign#saveOffline` removed, is there any functionality still being used from `offline-campaign` Stimulus controller? I get the impression that `offline-campaign` is superceded by the `setup-offline` controller, so we can get rid of that link-up here and the Stimulus controller.